### DISCUSSION TITLE
Need Help Verifying Student Status for GitHub Education

### DISCUSSION THREAD

**Discussion by Toisoi** [community/community#125019] [GitHub Education]
### Hi everyone,

I'm trying to sign up for GitHub Education, but I'm encountering a problem. The signup process requires a university email address, but my university doesn't provide student emails.
I do have a document confirming my student status (Temporary Residence Permit for Bulgaria), the "Remarks" section of this document explicitly states "student."

**Comment by davevad93** [comment]
Hi @Toisoi ,

Try taking a look at [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/#:~:text=If%20you%20don't%20have,verify%20your%20current%20student%20status.),  it's an old GitHub blog post from 2019 so I'm not sure if it's still valid. In case it isn't useful try contacting with GitHub support [here](https://support.github.com/request/education).

**Reply by Toisoi** [reply]
The problem is that after indicating the university, they do not allow me to go further after pressing the button. I'll write to technical support
![image](https://github.com/community/community/assets/63291809/450f4a30-47a1-4ad4-abbb-bfbd3e32b710)

**Reply by davevad93** [reply]
Hey @Toisoi  just as I imagined, yes, better contact support hopefully they can help you find a solution.
